example.env

```diff
@@ -0,0 +1,3 @@
+# copy and paste this to `.env`, then set values as appropriate
+adminUser="define your username"
+adminPassword="define your password"
```

Comment:
Only for discussion purpose but should we also include `mongoHost` (and remove the `environment` in the `docker-compose.yml`) so we consolidate our approach?

Reply:
I think so. As it would keep it consistent.